src/content-loader/index.js

```diff
@@ -31,11 +31,15 @@ module.exports = function(input) {
   const defaultHead = query.defaultHead
   const parsed = frontMatterParser(input)
 
+  // Normalize all data to JSON compaitable types
+  // See https://github.com/MoOx/phenomic/issues/397
+  const parsedData = JSON.parse(JSON.stringify(parsed.data))
```

Comment:
> This will slow the process. I would prefer another solution.

I agree.

> Maybe we should not export as json on the client side, since that not just json.

This is possible if we remove server side rendering, babel-plugin-webpack loader. I'm waiting for this.